BEIMA.Client/src/pages/Help/HelpPage.js

```diff
@@ -1,7 +1,7 @@
 import { Card } from "react-bootstrap";
 import { useOutletContext } from 'react-router-dom';
 import { useEffect } from 'react';
-import './HelpPage.css'
+import styles from'./HelpPage.module.css'
```

Comment:
Does this still compile properly without a space between "from" and the file path?

Reply:
I think the build.sh would have gotten angry if it didn't, so we should be good

Reply:
I'll update it regardless